force-app/main/default/classes/CheckFulfillmentOrders_Schedule.cls

```diff
@@ -0,0 +1,5 @@
+public class CheckFulfillmentOrders_Schedule implements Schedulable{
+    public void execute(SchedulableContext ctx) {
+        Database.executeBatch(new CheckFulfillmentOrders_Batch(), 5);
```

Comment:
Following the exercise this should be configurable, not fixed. You could create a new field on the Custom Metadata Type 'CheckFulfillmentOrderBatchSize' or something like this.